### PR TITLE
Feature/nav bar dynamic subcategories

### PR DESCRIPTION
**Pull request by tylerlam1**
refactor the category names

### CODE REVIEW

**Review by jcayabyab — COMMENTED** [review]
I thought we were dynamically updating the categories based on an API request

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -63,43 +66,70 @@ const SearchBar = styled.input`
...
+  const categories = useSelector(state => state.categories);
   const dispatch = useDispatch();
   const history = useHistory();
 
+  useEffect(() => {
+    const getAllCategories = async () => {
+      try {
+        dispatch(await getCategories());
+      } catch (e) {
+        error = e;
```
Comment (jcayabyab) [inline src/components/Navbar.jsx]: just one thing: Maybe make this a console.log - error is not defined here

**Review by jcayabyab — APPROVED** [review]

File: src/components/Navbar.jsx (hunk 1 above)
Comment (jcayabyab) [reply src/components/Navbar.jsx]: otherwise good to merge

Comment (tylerlam1) [reply src/components/Navbar.jsx]: ily

**Pull request merged**